src/CTA.Rules.ProjectFile/ProjectFileCreator.cs

```diff
@@ -133,16 +130,31 @@ public bool Create()
                 }
                 else
```

Comment:
I'm concerned about the if block right before this, if `IsCoreProject()` returns true then doesn't that mean that `CreateContents()` will return null? I'm not sure it matters for w2b but will this break normal CTA calls to `Create()`? I only ask because it seems like normally `Create()` would only return false in the event of an exception being thrown whereas that wouldn't hold in this case

Reply:
Hmm, I'm not to sure how to handle this, will prob discuss with you guys tomorrow